docs/src/installation/deployment.md

```diff
@@ -574,11 +574,68 @@ If you decided to set up a dex server, you can parametrize the [metal role](http
 
 ## Bootstrapping a Partition
 
-Documentation will follow soon. Please come to our Slack channel when you have come to this point and require any help.
+### Management network
 
-## Partition Deployment
+To be able to deploy and maintain a metal-stack partition, you need to bootstrap the management network first. 
+Some considerations must be made to fullfill the requirements of our infrastructure, a partition is designed to be:
+
+- secure
+- fully routable (BGP)
+- scalable
+- resilient
+- deployable via CI/CD jobs
+- accessible from the internet from specific IPs
+
+In order to accomplish this task remotely and in a nearly automatic manner, you have to bootstrap the components in this order:
+
+1. Management Firewalls
+2. Management Servers
+3. Management Spines
+4. Management Leaves
+5. Leaves, Spines and Exits
+
+This document assumes that all cabling is done. Here is a quick overview of the architecture:
+
+![Management Net](mgmt_net_layer3.svg)
```

Comment:
Is one image there broken? 

![Screenshot from 2021-02-10 08-16-55](https://user-images.githubusercontent.com/15035165/107477644-69c99c00-6b78-11eb-90e3-19dc3aaf1e75.png)

Reply:
converted to png